app/src/main/java/droiddevs/com/tripplanner/suggestedplaces/SuggestedPlacesPresenter.java

```diff
@@ -53,11 +54,11 @@ public void onFailure() {
     }
 
     private void loadSuggestedPlacesForDestination(Destination destination, String placeTypeSearchString) {
-        final String GoogleAPIKey = "AIzaSyBLM-VesFSnqP6wvQJFB0pxOmMX4QDHHig";
         final String locationString = String.valueOf(destination.getLatitude())
                 + "," + String.valueOf(destination.getLongitude());
 
-        mRepository.searchGooglePlaces(locationString, 3219, placeTypeSearchString, GoogleAPIKey,
+        // 3219 = 1 mile in meters
+        mRepository.searchGooglePlaces(locationString, 3219, placeTypeSearchString, TripPlannerApplication.getGooglePlacesApiKey(),
```

Comment:
Minor:
1 mile = 1,609 meters

PS 1 mile radius may be a bit narrow, you may want to test with a bigger radius here (i.e. 10 km), to allow for more places around a city's center